src/rdma.c

```diff
@@ -1184,11 +1184,16 @@ static int connRdmaConnect(connection *conn,
                            const char *addr,
                            int port,
                            const char *src_addr,
+                           int multipath,
                            ConnectionCallbackFunc connect_handler) {
     rdma_connection *rdma_conn = (rdma_connection *)conn;
     struct rdma_cm_id *cm_id;
     RdmaContext *ctx;
 
+    if (multipath) {
+        serverLog(LL_NOTICE, "RDMA: multipath not supported, ignore");
```

Comment:
I'm trying to understand what's best and the most logical choice here. We have three options:

1. Error, `assert(!multipath)`
2. Log notice
3. Silent

Do we use RDMA for outgoing connections at all?

I'm trying to think of some kind of future scenario where we use RDMA for some outgoing connections and MPTCP for some other connections and what should happen in this case.

For example, if we allow RDMA for replication, then we should reject the configuration `repl-mptcp yes` in the config validation. It means we still can't get to this case. If it can't happen, then maybe an assert is better?

Reply:
From the point of my view, assert is better at current stage.